web-components/src/components/table/scss/module.scss

```diff
@@ -0,0 +1,16 @@
+@import "@/wc_scss/tools/functions/core";
```

Comment:
add disclaimer here to. really, it should be in every file. for the purposes of this pr - add the disclaimer in every new file you are creating. both in component and sandbox

Reply:
Added